docs/Course Wiki/Taking X with Y/ECE210 with CS225.md

```diff
@@ -0,0 +1,18 @@
+# ECE210 with CS225
+
+## Relevant Course Reviews:
+- [ECE210](../ECE%20Course%20Offerings/ECE210.md)
+- [CS225](../CS%20Course%20Offerings/CS225.md)
+
+## Who should take them?
+Both courses are required for CompE majors, whereas [ECE210](../ECE%20Course%20Offerings/ECE210.md) is required for EEs and some choose to take [CS225](../CS%20Course%20Offerings/CS225.md) as an advanced core elective and/or for a CS minor. Both courses are generally taken sophomore year, and they do not relate very much. ECE210 focuses on circuit analysis concepts and basics of signal processing such as frequency response, Fourier transforms, and convolution, whereas CS225 focuses on how to store data using data structures such as stacks, queues, and trees and is taught using C++.
+
+## Workload:
+In terms of coursework, [ECE210](../ECE%20Course%20Offerings/ECE210.md) has a weekly homework assignment, biweekly labs starting halfway through the semester, 3 midterms, and a final. Homework assignments usually contain around 5-7 problems that need to be solved. For labs, there is usually a 2-page prelab and then a 4-page lab report alongside the required circuit construction. One can expect to spend around 3-6 hours on these assignments.
+
+Meanwhile, [CS225](../CS%20Course%20Offerings/CS225.md) has 6 MPs occuring biweekly, a weekly lab, 6 exams, and a final. MPs are usually divided into 2 parts, where the first part can be completed a week early for extra credit, while the whole MP is due all at once after the 2 weeks are up. Weekly labs can usually be done during the lab sections and are similar in length to [ECE220](../ECE%20Course%20Offerings/ECE220.md) MPs, while the MPs are usually much larger in scale. One can expect to spend around 10-15 hours per week on these assignments. There is also an optional group-based final project for extra credit, which some students may choose to pursue for various reasons.
+
+## Recommendation:
+Yes, taking these two courses together is very much doable. Since these two courses are not related, they can be and commonly are taken together, especially by CompE students who took [ECE110](../ECE%20Course%20Offerings/ECE110.md) and [ECE220](../ECE%20Course%20Offerings/ECE220.md) together.
```

Comment:
Maybe add something how the main focus of 210 is the exams, whereas the exams in 225 are much easier and more emphasis is put on the MPs